docs/usage/merge-confidence.md

```diff
@@ -25,11 +25,15 @@ Merge Confidence badges for pull requests are available on any supported platfor
 
 ## Supported languages
 
-Data is available for packages from:
-
-- npm
-- Maven
-- PyPI
+Data is available for the following languages (from the noted data source):
+
+- JavaScript (npm)
+- Java (maven)
+- Python (pypi)
+- .NET (nuget)
+- PHP (packagist)
+- Ruby (rubygems)
+- Scala (sbt-package)
```

Comment:
Are you sure this will work?

Reply:
We should automate this list somehow, and the code should be the source of truth for this list.

How about putting a `showsMergeConfidenceBadges` Boolean in the datasource code? Then we can:

- create a list of all datasources that have `showsMergeConfidenceBadges= true`
- when creating the datasource docs, show a new item "Shows Merge Confidence badges: yes/no" with a link to the `merge-confidence.md` file

Reply:
It's nice to have, but it's a simple list so I also don't mind if we keep it manual. I certainly wouldn't hold up this feature waiting for it

Reply:
@rarkins do you want an issue from me for this "feature request" then? Or do you want to keep it manual?

Reply:
Actually I think it's ok to keep manual, once it's confirmed it's correct

Reply:
Removed Scala from Merge Confidence documentation.